.github/workflows/lint.yml

```diff
@@ -35,3 +35,11 @@ jobs:
         uses: ludeeus/action-shellcheck@master
         with:
           scandir: './scripts'
+  linter_name:
+    name: runner / black formatter
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: rickstaa/action-black@v1
```

Comment:
Do we need to use this action or is the [official psf/black action](https://black.readthedocs.io/en/stable/integrations/github_actions.html) good enough for us?

Reply:
No of course we use the offical one. I thought I had copy & pasted the quickstart from the offical one.